.gitignore

```diff
@@ -3,3 +3,4 @@
 target/
 env/
 *.pyc 
+docker-compose.prod.yml
```

Comment:
Thinking out loud: why is a file - essential for a specific environment - excluded from the repository?

Reply:
That file's never supposed to on the public repository as it can contain secret information that could compromise a live server.